src/handlers/motions/helpers.ts

```diff
@@ -38,19 +38,10 @@ export const getMotionFromDB = async (
     },
   );
 
-  const motionsWithCorrectId = motions.filter(
-    ({ motionData: { motionId: id } }) => id === motionId.toString(),
+  const motion = motions.find(
```

Comment:
So I think I just left a comment about refactoring this in #32 

Feel free to ignore that then :)